rosbag2_storage/test/rosbag2_storage/test_read_only_plugin.cpp

```diff
@@ -54,6 +54,12 @@ std::vector<rosbag2_storage::TopicMetadata> TestReadOnlyPlugin::get_all_topics_a
   return std::vector<rosbag2_storage::TopicMetadata>();
 }
 
+std::string TestReadOnlyPlugin::get_relative_path() const
+{
+  std::cout << "\nreturning relative path\n";
+  return "";
```

Comment:
does it make sense to return a dummy path in here to test for?

Reply:
nice catch. It should so we can test it